src/herbivoresAndCarnivores.js

```diff
@@ -1,15 +1,39 @@
 'use strict';
 
 class Animal {
-  // write your code here
+  static alive = [];
+
+  health = 100;
+
+  constructor(name) {
+    this.name = name;
+
+    Animal.alive.push(this);
+  }
 }
 
 class Herbivore extends Animal {
-  // write your code here
+  hidden = false;
+
+  hide() {
+    this.hidden = true;
+  }
 }
 
 class Carnivore extends Animal {
-  // write your code here
+  bite(someHerbivore) {
+    if (
+      typeof someHerbivore === 'object' &&
```

Comment:
Maybe you don't need to check if it's an object, if you already check if it's an instance of the herbivore class, though I'm not sure.

Reply:
you 100% rigth, i should delete it, instance is object